great_expectations/render/components.py

```diff
@@ -454,9 +472,24 @@ def to_json_dict(self) -> dict[str, JSONValues]:
         return d
 
 
+@public_api
 class RenderedStringTemplateContent(RenderedComponentContent):
+    """RenderedStringTemplateContent is RenderedComponentContent that represents a templated string.
+
+    Args:
+        string_template: A dictionary containing:
+            template: The string to perform substitution on. Variables are denoted with a preceeding $.
+            params: A dictionary with keys that match variable names and values which will be substituted.
+            styling: A dictionary containing styling information.
```

Comment:
Whitespace might help with the rendering issues. Trying to test myself locally but running into other issues. This goes for all the nested args
![image](https://user-images.githubusercontent.com/9903066/215569485-7c8ab359-5a19-4d7a-91fd-b6376a82a01c.png)

```suggestion
        string_template: A dictionary containing:

            template: The string to perform substitution on. Variables are denoted with a preceeding $.

            params: A dictionary with keys that match variable names and values which will be substituted.

            styling: A dictionary containing styling information.
```